CITATION.cff

```diff
@@ -4,7 +4,7 @@ authors:
   - family-names: "Kollitsch"
     given-names: "Patrick"
     orcid: "https://orcid.org/0000-0001-6250-2135"
-title: "DNB Org Hugo Components / Search Algolia"
+title: "David's Neighbour Org Hugo Components / Search Algolia"
```

Comment:
I would call them "Hugo Components / Component Name" to take the whole "vanity" part out of it. The citation file is for citing setup and the title doesn't need to be overly long. If you enable edits by maintainers then I'll change that before merging your PRs in all other projects.

Reply:
@davidsneighbour I think I fixed them all. ;-)